autoendpoint/src/extractors/subscription.rs

```diff
@@ -35,6 +35,10 @@ pub struct Subscription {
     pub user: User,
     pub channel_id: Uuid,
     pub vapid: Option<VapidHeaderWithKey>,
+    /// Should this subscription update be tracked internally?
+    /// (This should ONLY be applied for messages that match known
+    /// Mozilla provided VAPID public keys.)
+    pub tracking_id: Option<String>,
```

Comment:
This was changed from a simple bool to an `Option<String>` since that's most likely the final form of this. For now, we're just using a "filler" UUID4, but I strongly suspect that https://mozilla-hub.atlassian.net/browse/SYNC-3400 will introduce a more formalized tracking_id generator.

Reply:
Seems like the comment still reflects the bool -- so it needs an update

Reply:
Well, it _is_ an option, so it has bool like properties (None vs. Some). I'll clarify the comment.